src/components/PaginatedTable/PaginatedTable.tsx

```diff
@@ -37,7 +37,7 @@ interface Filter {
 
 interface PagedResults<EntryT = any> {
     count: number; // total results
-    results: Array<any>;
+    results: Array<EntryT>;
 }
 
 type SourceFunction<EntryT> = (
```

Comment:
Just guessing here, but should `sorting` also have type `Array<EntryT>` below or is it correct as is?

Reply:
I think it's correct as-is.  `order_by` (a `string[]`) is passed as `sorting` when `props.source` is called. (`order_by` specifies the ordering for the table. example: on the [**Puzzles**](https://online-go.com/puzzles) page, if I sort on the rank column, `order_by = ['min_rank', 'max_rank']`)

https://github.com/online-go/online-go.com/blob/29b7a1160d6634fa66c9fa0f3b93d201f7ee8b79/src/components/PaginatedTable/PaginatedTable.tsx#L134

The funny thing though is looking into it, I couldn't find any examples of `SourceFunction` in action.  It seems like all existing `PaginatedTable`s have `props.source` set as a string, not a function.  Perhaps the function option is just used for testing.  @anoek may be able to give insight.

Reply:
Yeah it doesn't look like we ever use source as a function. Could have been that we did once upon a time, or that I thought we might when I was originally implementing that - but we don't, and it doesn't seem particularly useful so I'd say it's safe to remove and just use strings if that makes things easier.

Reply:
Is this what you had in mind: https://github.com/online-go/online-go.com/pull/1641/commits/d80902e70f28ad4c1c40e612bc3abe69fdac73cf? If so looks a lot simpler to me.